2-mandatory/1-create-functions.js

```diff
@@ -33,7 +36,8 @@ Write a function that:
 - Returns a new array containing the same elements, but without the element at the passed index.
 */
 
-function remove() {
+function remove(arr, i) {
+  return arr.filter(e => e != arr[i]);
```

Comment:
What would happen here if the input was: `remove(["hello", "world", "hello"], 2)`?

Reply:
You are right, this is a bug, I already changed the solution